doc/user_manual/include/basicEventScheduler.tex

```diff
@@ -0,0 +1,41 @@
+\section{Basic Event Scheduler Model}
+\label{sec:basicEventScheduler}
+
+This model is designed to read the initial and final time under which a set of basic events
+are set to True, and then it generates an HistorySet with the full temporal profile of all
+basic events.
```

Comment:
I recommend to add another paragraph to explain the structure of generated history set.